src/mito2/src/config.rs

```diff
@@ -56,6 +56,10 @@ pub struct MitoConfig {
     pub global_write_buffer_size: ReadableSize,
     /// Global write buffer size threshold to reject write requests (default 2G).
     pub global_write_buffer_reject_size: ReadableSize,
+
+    // Cache configs:
+    /// Total size for cache (default 512MB). Setting it to 0 to disable cache.
+    pub cache_size: ReadableSize,
```

Comment:
May be changed to `sst_cache_size`? There will be other cache types in future ,such as indexing cache, query cache etc.

Reply:
I plan to share the same cache for different data, so users only need to configure one cache size. Maybe we should cache different data separately and have different cache limits for them.

I will change this to `sst_meta_cache_size`.

Reply:
One cache configuration to rule everything is a good idea.

Reply:
It's more flexible to allow setting size limits for different data. We can also disable particular cache.